### DISCUSSION TITLE
Magic link authentication, transactional emails

### DISCUSSION THREAD

**Discussion by richcorbs** [pockethost/pockethost#156] [Q&A]
Loving pocketbase and pockethost so far!

I have a few email-related ideas to ask about, none are showstoppers and none may even be good ideas :-).

(1) When sending emails from subdomains is possible, would it also be possible to add magic link authentication as an option?

(2) Relatedly, could a transactional email feature be added where we can create and send/trigger transactional emails via the api?

(3) When sending emails from subdomains is possible, will it be possible to set up the sending email to look like `My App <subdomain@pockethost.io>` when it arrives in the inbox? And could the "Reply To" address be customized?

**Comment by benallfree** [comment]
Hi @richcorbs, I'll see if I can answer these or give some thoughts :)

1. Are you talking about magic links for users of your application? (Not PocketHost, but registered users of your instance?). If so, I think that would be more of a PocketBase feature. Check out https://github.com/pocketbase/pocketbase/issues/119 for more details.
2. This may also be more of a core PocketBase feature, but I'm considering it. Follow https://github.com/pocketbase/pocketbase/discussions/1321 for more info on PocketBase updates.
3. Yes, #24 is where this planning is taking place. Feel free to chime in. What you just described is very close to how I see it working too.
